gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -93,14 +104,29 @@ class Async{{service.name}}RestTransport(_Base{{ service.name }}RestTransport):
     {{ shared_macros.wrap_async_method_macro()|indent(4) }}
 
     {% for method in service.methods.values()|sort(attribute="name") %}
+    class _{{method.name}}(_Base{{ service.name }}RestTransport._Base{{method.name}}, Async{{service.name}}RestStub):
+        def __hash__(self):
+            return hash("Async{{service.name}}RestTransport.{{method.name}}")
+
+        async def __call__(self,
+                    request: {{method.input.ident}}, *,
```

Comment:
What does the asterisk mean here?

Reply:
arguments following `*` must be passed in as keyword arguments.

Reply:
Oh, it's a Python thing. Because it was on the same line, I assumed it was part of the type hint and glossed over the comma.

I suggest having the `*` in its own line, like the other parameters. It gets lost here.